tests/search_gov_spiders/test_full_crawl.py

```diff
@@ -101,6 +102,7 @@ def mock_init(pipeline_cls, *_args, temp_dir=temp_dir, **_kwargs):
             pipeline_cls.short_file = open(pipeline_cls.base_path_name, "w", encoding="utf-8")
             pipeline_cls.max_file_size = 3900
             pipeline_cls.paginate = True
+            pipeline_cls.lock = threading.Lock()
```

Comment:
i don't think we want this in the test anymore, right?

Reply:
isn't it a totally different library?  there isn't even a `lock` instance variable in the actual code.